src/components/override-sdk/infra/ActionButtons/ActionButtons.tsx

```diff
@@ -4,13 +4,29 @@ import Button from '../../../BaseComponents/Button/Button';
 import { useTranslation } from 'react-i18next';
 
 export default function ActionButtons(props) {
-  const { arMainButtons, arSecondaryButtons, onButtonPress, isUnAuth, isHICBC } = props;
+  const { arMainButtons, arSecondaryButtons, onButtonPress, isUnAuth, isHICBC, getPConnect } =
+    props;
   const localizedVal = PCore.getLocaleUtils().getLocaleValue;
   const localeCategory = 'Assignment';
+  const taskListStepId = 'SubProcessSF7_AssignmentSF1';
```

Comment:
Is it safe to hard-code this value?

Reply:
As discussed, this is fine for tactical solution as pega want to hard code this value from react.